resources/gcp/compute/vm.go

```diff
@@ -0,0 +1,57 @@
+package compute
+
+import (
+	"github.com/DataDog/test-infra-definitions/common/config"
+	"github.com/DataDog/test-infra-definitions/common/utils"
+	"github.com/DataDog/test-infra-definitions/resources/gcp"
+	"github.com/pulumi/pulumi-gcp/sdk/v7/go/gcp/compute"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func NewLinuxInstance(e gcp.Environment, name string, imageName string, instanceType string, opts ...pulumi.ResourceOption) (*compute.Instance, error) {
+
+	sshPublicKey, err := utils.GetSSHPublicKey(e.DefaultPublicKeyPath())
+	if err != nil {
+		return nil, err
+	}
+	vmOpts := 
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Aren't you missing something?

Reply:
oops. Need to remove that